api/api.go

```diff
@@ -30,6 +30,9 @@ const (
 
 	// PathDohQuery DoH Url
 	PathDohQuery = "/dns-query"
+
+	// HealthPath Url
+	HealthPath = "/health"
```

Comment:
I think using `/healthz` would be nice as it's the usual convention for docker and kubernetes.

Reply:
Sounds sensible to me, thoughts @0xERR0R ?

Reply:
I think, all REST endpoints should start with "/api". But we have already a special endpoint "/dns-query" (because it is the convention) and healtcheck is also something special. I think all 3 variants are Ok.